src/main/java/org/beanpod/switchboard/controller/LogController.java

```diff
@@ -39,17 +39,20 @@
 
   @Override
   public ResponseEntity<List<StreamLogModel>> retrieveStreamLogs(@PathVariable Long streamId) {
-    return ResponseEntity.ok(streamLogDao.getStreamLogs(streamId));
+    UserEntity user = userDao.findUser(request.getUserPrincipal().getName());
+    return ResponseEntity.ok(streamLogDao.getStreamLogs(streamId, user.getId()));
   }
```

Comment:
retrieving a stream log returns an empty list
creating a regular log results in an error for me

Reply:
Let me know what steps you're taking because on my end I haven't encountered an error??? I'll double check

Reply:
Retrieving a stream log gives internal server error for me... i thought i tested it but i guess not

Reply:
My steps to reproduce are
- wipe the database
- compile and run the backend
- register sample sender and sample receiver
- create stream between the two devices in the frontend
- use postman to call GET /log/stream/1
```
{
    "status": "INTERNAL_SERVER_ERROR",
    "message": "could not extract ResultSet; SQL [n/a]; nested exception is org.hibernate.exception.SQLGrammarException: could not extract ResultSet",
    "error": "An unknown error occured"
}
```

Reply:
I see 🤔 ... I was running the frontend and things seemed fine. I wiped the DB and tested using postman. I got this error for regular log
```
{
    "status": "INTERNAL_SERVER_ERROR",
    "message": "id is marked non-null but is null",
    "error": "An unknown error occured"
}
```

Reply:
Thanks for catching this! It's resolved right now.